doc/serializers.rst

```diff
@@ -1,92 +1,61 @@
 Serializers
 ===========
 
-Bernard supports multiple serializers to serialize messages for persistent
-storage, such as Predis, Redis and so on. So far, the examples have been using
-JMS as it was the first supported.
+Bernard uses the `Symfony Serializer Component <http://symfony.com/doc/current/components/serializer.html>`_ to
+serialize messages for persistent storage.
 
-The JMS Serializer is only recommended if you already use this serializer or
-if you need advanced serialization in your application.
+Default serializer
+------------------
 
-Naive
------
-
-Bernard ships with a Naive serializer, which only supported using
-``DefaultMessage`` messages. And as such it should only be used when starting up
-or playing with Bernard.
+By default Bernard can handle serializing the ``Bernard\Envelope`` and ``Bernard\Message\DefaultMessage`` classes,
+which should be enough when you are just starting out:
 
 .. code-block:: php
 
     <?php
 
-    use Bernard\Serializer\NaiveSerializer;
-
-    $serializer = new NaiveSerializer();
-
+    use Bernard\Driver\FlatFileDriver;
+    use Bernard\QueueFactory\PersistentFactory;
+    use Bernard\Serializer;
```

Comment:
This should be simplified to not have any knowledge of the Queues and Drivers.

Something like: 

``` php
<?php

use Bernard\Serializer;

$serializer = new Serializer()
$serializer->serialize($envelope)
```